### ISSUE TITLE
Fix S3264: FP when using add/remove syntax

### ISSUE THREAD

**Issue by Evangelink** [SonarSource/sonar-csharp#1219]
### Description

Rule raises FP when using the add/remove syntax of an `event`.

### Repro steps

```
public class Foo
{
    private EventHandler someEvent;
    public event EventHandler SomeEvent // Noncompliant - should be compliant
    {
        add
        {
            someEvent += value;
        }
        remove
        {
            someEvent -= value;
        }
    }

    public DoRaise()
    {
        someEvent?.Invoke(this, EventArgs.Empty);
    }
}
```

### Expected behavior

Issue not raised because the event is raised using the private handler.

### Actual behavior

Issue is raised because we don't use the `event` `Invoke` whereas it is impossible to do so.

### Known workarounds

None.

### Related information

* SonarC# Version 7.0
* Visual Studio Version 2015/2017
